obs-package-status/Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gem 'nokogiri'
+gem 'pry'
```

Comment:
I think we can remove this gem, or is for the future if we need to debug?

Reply:
yeah I don't want to install it manually all the time...